utils/mqttled/files/mqttled.init

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+STOP=99
+
+USE_PROCD=1
+
+PROG=/usr/bin/mqttled
+
+start_service() {
+	procd_open_instance
+	procd_set_param command "$PROG"
+	procd_set_param stdout 1
+	procd_set_param stderr 1
+	procd_close_instance
+}
+
+stop_service() {
```

Comment:
is this section even needed?

Reply:
Fair enough, just copied another init script I found: removed